schema/controller/cluster_backup.json

```diff
@@ -1,11 +1,52 @@
 {
   "$schema": "http://json-schema.org/draft-04/schema#",
   "id": "https://flynn.io/schema/controller/cluster_backup#",
-  "title": "Backup",
+  "title": "Cluster Backup",
   "description": "",
   "sortIndex": 18,
-  "type": "string",
+  "type": "object",
   "examples": [
     "schema/examples/controller/cluster_backup#"
-  ]
+  ],
+  "definitions": {
+    "backup_status": {
+      "type": "string",
+      "enum": ["running", "complete", "error"]
+    },
+    "completed_at": {
+      "description": "timestamp of backup completion",
+      "format": "date-time",
+      "type": "string"
+    }
+  },
+  "additionalProperties": false,
+  "properties": {
+    "id": {
+      "$ref": "/schema/controller/common#/definitions/id"
+    },
+    "status": {
+      "$ref": "#/definitions/backup_status"
+    },
+    "sha512": {
+      "description": "base64 encoded sha512 of backup",
+      "type": "string",
+      "format": "base64"
```

Comment:
I think we should hex-encode the hash as all standard tools work with hex hashes, not base64.